css/styles.css

```diff
@@ -184,9 +187,95 @@ cursor: pointer;
   font-size:25px;
   }
 }
-}
 @media screen and (max-width: 290px){
   input[type=submit]{
     width:30%;  
  }
+}
+/************skills***************/
+ .skills{ 
+  margin-left:300px;
+  text-align: center;
+  padding:10% 0;
+}
+.row{
+  display: flex;
+  flex-direction: row;
+  padding:5%;
+  justify-content: space-between;
+  border-bottom: 0.02px solid white;
```

Comment:
@unadine, I think this border-bottom is supposed to be in the `skills` instead of here because this line indicates that the section end.